app/src/main/java/org/p2p/wallet/auth/ui/restore/common/CommonRestoreFragment.kt

```diff
@@ -43,8 +45,15 @@ class CommonRestoreFragment :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         with(binding) {
-            toolbar.setNavigationOnClickListener {
-                popBackStack()
+            toolbar.setNavigationOnClickListener { popBackStack() }
+            toolbar.setOnMenuItemClickListener {
+                if (it.itemId == R.id.helpItem) {
+                    // pass empty string as UserId to launch IntercomService as anonymous user
+                    IntercomService.signIn(userId = emptyString())
```

Comment:
Intercom is signed in with empty user in OnboardingRootPresenter

Reply:
are you sure?
need to check
but if it is true so we do not need to signIn any more?

Reply:
We should have userId on Derivation path screen
You're getting in when choosing the path, right?

Reply:
1. We have empty user in RootPresenter, I added it lately 
2. Indeed, we already have an userId (publicKey) on the restore wallet derivation step